context-config-store/store/persistence/default_context_config.go

```diff
@@ -0,0 +1,10 @@
+package persistence
+
+import api "github.com/kurtosis-tech/kurtosis/context-config-store/api/golang"
+
+var personalContextName = "personal"
+var personnalContextUuid = api.NewContextUuid("00000000-0000-0000-0000-000000000000")
```

Comment:
I suggest we change that for `default` and also planning to using our own uuid generator, so UUID will be `00000000000000000000000000000000` instead

Reply:
Updated the notion doc already, will vet that to Kevin just in case